suite/tests/api/ir_x86.c

```diff
@@ -266,12 +267,12 @@ test_all_opcodes_2_avx512_evex(void *dc)
 }
 
 static void
-test_all_opcodes_2_avx512_evex_disp8(void *dc)
+test_all_opcodes_2_avx512_evex_comprdisp8(void *dc)
```

Comment:
Not easy to read "comprdisp".  How about scaled_disp8 or sthg.

Reply:
Done.